notebooks/pytorch_sparse_quantized_transfer_learning.ipynb

```diff
@@ -0,0 +1,460 @@
+    "    optimizer,\n",
+    "    model,\n",
+    "    manager,\n",
+    "    steps_per_epoch=len(train_loader),\n",
+    "    loggers=[],\n",
+    ")\n",
+    "\n",
+    "\n",
+    "# Run model pruning\n",
+    "epoch = manager.min_epochs\n",
+    "for epoch in range(manager.max_epochs):\n",
+    "    # run training loop\n",
+    "    epoch_name = f\"{epoch + 1}/{manager.max_epochs}\"\n",
+    "    print(f\"Running Training Epoch {epoch_name}\")\n",
+    "    train_loss, train_acc = run_model_one_epoch(\n",
+    "        model, train_loader, criterion, device, train=True, optimizer=optimizer\n",
+    "    )\n",
+    "    print(\n",
+    "        f\"Training Epoch: {epoch_name}\\nTraining Loss: {train_loss}\\nTop 1 Acc: {train_acc}\\n\"\n",
+    "    )\n",
+    "\n",
+    "    # run validation loop\n",
+    "    print(f\"Running Validation Epoch {epoch_name}\")\n",
+    "    val_loss, val_acc = run_model_one_epoch(model, train_loader, criterion, device)\n",
+    "    print(\n",
+    "        f\"Validation Epoch: {epoch_name}\\nVal Loss: {val_loss}\\nTop 1 Acc: {val_acc}\\n\"\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 6 - View Model Sparsity\n",
+    "To see the effects of sparse-quantized transfer learning, in this step, you will print out the sparsities of each Conv and FC layer in your model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sparseml.pytorch.utils import get_prunable_layers, tensor_sparsity\n",
+    "\n",
+    "# print sparsities of each layer\n",
+    "for (name, layer) in get_prunable_layers(model):\n",
+    "    print(f\"{name}.weight: {tensor_sparsity(layer.weight).item():.4f}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 7 - Exporting to ONNX\n",
+    "\n",
+    "Now that the sparse-quantized transfer learning is complete, it should be prepped for inference.  A common next step for inference is exporting the model to ONNX.  This is also the format used by the [DeepSparse Engine](https://github.com/neuralmagic/deepsparse) to achieve the sparse-quantized speedups.\n",
+    "\n",
+    "For PyTorch, exporting to ONNX is natively supported. In the cell block below, a convenience class, ModuleExporter(), is used to handle exporting.\n",
+    "\n",
+    "Additionally, PyTorch, exports a graph setup for quantization aware training (QAT) to ONNX. To run a fully quantized graph, you will need to convert these QAT operations to fully quantized INT8 operations.  SparseML provides the `quantize_torch_qat_export` helper function to perform this conversion.\n",
+    "\n",
+    "Once the model is saved as an ONNX ﬁle, it is ready to be used for inference with the DeepSparse Engine.  For saving a custom model, you can override the sample batch for ONNX graph freezing and locations to save to.\n",
```

Comment:
end of the "For saving" sentence sounds a little awkward. I would do something like
For saving a custom model, you can override two settings (? or whatever they are called here): the sample batch number (?) for ONNX graph freezing and the model's file path destination (?).

Reply:
on second read, we actually are taking out custom instructions from this nb, so just going to take the sentence out